.github/workflows/csound_builds.yml

```diff
@@ -101,6 +101,53 @@ jobs:
           name: iOS-builds
           path: iOS/csound-iOS-7.0.0 # or path/to/artifact
 
+  ios_build_ioscross:
+    name: iOS build (ioscross/vcpkg)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - uses: lukka/get-cmake@v3.29.0
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Bootstrap VCPKG
+        run: ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install build-essential liblzma-dev libxml2-dev libz-dev libbz2-dev cpio ninja-build
+
+      - name: Download ioscross
+        run: |
+          curl -L "https://drive.usercontent.google.com/download?id=1COyrQ7ypTTVBiI9dBIOvswIm1CWbNFa0&confirm=xxx" | tar xvz
+
+      - name: Configure Build
+        run: |
+          $(eval /home/runner/work/csound/csound/ioscross/ioscross_conf.sh)
+          cmake -B build/arm64 -S . -DUSE_VCPKG=1 -DCUSTOM_CMAKE=./platform/ioscross/custom-ios.cmake \
+              -DIOS=1 \
```

Comment:
Maybe put all these extra flags inside the `custom-ios.cmake` ? Can clean this up a bit

Reply:
I moved as many of these flags as I could into the custom cmake files.  I wasn't able to move all of them since the vcpkg config happens before the custom cmake file is read.

Reply:
You can add platform specific code to where the VCPKG code is in the main cmakelists.txt as well